pubspec.yaml

```diff
@@ -148,6 +148,11 @@ msix_config:
   install_certificate: false
 
 dependency_overrides:
+  desktop_drop:
+    git:
+      url: https://github.com/TheOneWithTheBraid/flutter-plugins.git
+      path: packages/desktop_drop
+      ref: braid/unsupported-uri-scheme-callback
```

Comment:
dependency_overrides need a comment with a linked issue or MR when to solve this